### PR TITLE
Adding basic tests and so on

### PR DESCRIPTION
**Pull request by ikedas**
- Adding some basic test cases
  - Updating MANIFEST & MANIFEST.SKIP

N.B.: To make a new package, run
``` bash
$ make distclean
$ perl Makefile.PL
$ make
$ make disttest
$ make dist
```

### CODE REVIEW

File: t/compile.t [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+BEGIN {
+    eval 'use Test::Compile';
+    $Test::Compile::VERSION or plan skip_all => 'Test::Compile required';
+}
+
+grep { pl_file_ok $_ }
+    qw{mha-dbedit mha-dbrecover mha-decode mhonarc},
+    qw{admin/mhaadmin.cgi extras/mha-mhedit/mha-mhedit},
+    all_pl_files(qw{contrib examples lib}),
```
Comment (ldidry) [inline t/compile.t]: [Test::Compile documentation](https://metacpan.org/pod/Test::Compile#all_pl_files(@dirs)) says

> all_pl_files(@dirs)
>    This function is deprecated. Please use "all_pl_files(@dirs)" in Test::Compile::Internal instead.

Same thing for `all_pm_files(@dirs)`.

We should use [Test::Compile::Internal](https://metacpan.org/pod/Test::Compile::Internal)

**Comment by ikedas** [comment]
Comes under #10.

File: t/compile.t (hunk 1 above)
Comment (ldidry) [reply t/compile.t]: Sorry, I wrote this comment a while ago, but it was `Pending` (I’m not familiar with Github reviews workflow).

Comment (ikedas) [reply t/compile.t]: Thanks for comments. Fixed.

**Pull request merged**